Source/IGListAdapter.m

```diff
@@ -308,6 +308,16 @@ - (NSUInteger)sectionForSectionController:(IGListSectionController <IGListSectio
     return [self.sectionMap sectionControllerForObject:object];
 }
 
+- (id)objectForSectionController:(IGListSectionController <IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSUInteger section = [self.sectionMap sectionForSectionController:sectionController];
+    IGAssert(section != NSNotFound, @"Section nil for section controller %@", sectionController);
```

Comment:
I based this off the implementation for `-[IGListAdapter objectAtSection:]`, which crashes if you pass in an index that's out of bounds. The assert I wrote actually wouldn't work, but there definitely needs to be a bounds check.

I'm assuming that's another issue that should be fixed?